fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfiguration.java

```diff
@@ -92,6 +92,9 @@
     public static final String PROPERTY_DATASOURCES = "fhirServer/persistence/datasources";
     public static final String PROPERTY_JDBC_BOOTSTRAP_DB = "fhirServer/persistence/jdbc/bootstrapDb";
     public static final String PROPERTY_JDBC_DATASOURCE_JNDINAME = "fhirServer/persistence/jdbc/dataSourceJndiName";
+    public static final String PROPERTY_JDBC_BOOTSTRAP_DATASOURCE_BASE = "fhirServer/persistence/jdbc/bootstrapDataSourceBase";
```

Comment:
name is kind of wonkey

Reply:
No, it aligns with the existing PROPERTY_JDBC_BOOTSTRAP_DB. Intentional. I have moved it up one line to make this clear.